launch/mobileye_eval.launch

```diff
@@ -0,0 +1,57 @@
+<!--
```

Comment:
I am not sure where we should keep this file. Feel like does not belong this repo. Do you think it makes sense to put it in our engineering tool folder?

Reply:
I think it is best if we keep it with a package so we can run 
roslaunch <pkg> mobileye_eval.launch easily. 

I could move it to the CARMAPlatform repo if you want. There are some other testing launch files I have used in the past there. Let me know.